api/app/Console/Kernel.php

```diff
@@ -20,12 +22,25 @@ protected function schedule(Schedule $schedule)
     {
         // $schedule->command(HardDeleteOldUsers::class)->dailyAt('08:00');
 
-        // clean up old user generated files every day at 2:00 AM
+        // clean up old user generated files every day at 1:00 AM
         $schedule->command(PruneUserGeneratedFiles::class)
             ->timezone('America/Toronto')
-            ->dailyAt('2:00')
+            ->dailyAt('1:00')
             ->withoutOverlapping()
             ->appendOutputTo('/tmp/laravel-prune-user-generated-files.log');
+
+        if (config('feature.notifications')) {
+
+            // queue up Application Deadline Approaching emails every day, close to the time the pool would close
+            $schedule->command(SendNotificationsApplicationDeadlineApproaching::class)
+                ->timezone('America/Toronto')
+                // 10 PM Eastern is the same day across the country, close to the end of the day in NL
+                ->dailyAt('22:00')
+                ->appendOutputTo('/tmp/send-notifications-application-deadline-approaching.log');
```

Comment:
oh ..it logs in here .. nvm then. Does it have failures too?

Reply:
Good idea!  I've added that in [log errors](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10230/commits/accccd441d8ecf28e68c671eba78ebdfd6ee2a6d).
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a89411ec-50a8-49f5-85b6-56434d8c801c)